doc/api/cli.md

```diff
@@ -1171,3 +1180,4 @@ greater than `4` (its current default value). For more information, see the
 [experimental ECMAScript Module]: esm.html#esm_resolve_hook
 [libuv threadpool documentation]: http://docs.libuv.org/en/latest/threadpool.html
 [remote code execution]: https://www.owasp.org/index.php/Code_Injection
+[Subresource Integrity]: https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity
```

Comment:
This needs to be placed after `[ScriptCoverage]:`, alphabetically.

Reply:
these don't seem to be in alphabetical order though?

Reply:
Bottom references are usually sorted in ASCII order, case and punctuation are taken into account.